src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,30 @@
+package lotto.domain
+
+data class WinningLotto(
+    val lottoNumbers: LottoNumbers,
+    val bonus: LottoNumber,
+) {
+    init {
+        require(!lottoNumbers.containsNumber(bonus)) { "보너스 볼 번호는 로또 번호와 중복될 수 없습니다" }
```

Comment:
```suggestion
        require (bonus !in lottoNumbers) { "보너스 볼 번호는 로또 번호와 중복될 수 없습니다" }
```
operator fun contains를 활용한다면 이렇게도 구현이 가능해요!

Reply:
오,,! 코틀린스러워요 적용해보겠습니다👍